include/shellstrings.h

```diff
@@ -0,0 +1,128 @@
+
+/* 
+ * String output functions to be used in the shell
+*/
+
+#ifndef INCLUDE_SHELLSTRINGS_H_
+#define INCLUDE_SHELLSTRINGS_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*
+ * shell_prompt - Default text input prompt string for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_prompt();
+
+/*
+ * shell_intro - Introduction text for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_intro();
+
+/*
+ * shell_save - Modification saving text for the shell.
+ *
+ * Parameters:
+ *  - None.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_save();
+
+/*
+ * shell_print - Print edited text for viewing
+ *
+ * Parameters:
+ * 	- None.
+ *
+ * Returns:
+ * 	- None.
+ */
+ void shell_print(char** lines);
+
+
+/*
+ * shell_error - Default error prompt string for the shell.
+ *
+ * Parameters:
+ *  - error_text: String for the specific error text.
+ *
+ * Returns:
+ *  - None.
+ */
+void shell_error(char* error_text);
+
+/*
+ * shell_input - Input file confirmation prompt for the shell.
+ *
+ * Parameters:
+ *  - input_file: String of the file input by the user.
+ *  - mode: String for the mode being used (either quiet or verbose) 
```

Comment:
Why is verbosity a string here? This is the third strategy I've seen so far for specifying verbosity. Please `#define` your own flags and be consistent.

Reply:
This was mislabeled; this is not a mode, it's a file type (dictionary or input file). Correction's been made.